source/stylist/fortran.py

```diff
@@ -204,6 +204,35 @@ def examine_fortran(self, subject: FortranSource) -> List[Issue]:
         return issues
 
 
+class IntrinsicModule(FortranRule):
+    """
+    Catches cases where an intrinsic module is used with the "intrinsic"
+    keyword ommitted.
+    """
+
+    def __init__(self):
+        """
+        Constructs an "IntrinsicModule" rule object.
+        """
+        self.intrinsics = ["iso_c_binding", "iso_fortran_env",
```

Comment:
Convention is for constants, which this should be, to be named all-caps. It isn't magic, it doesn't make things constant but it is a notice of intent. Similar to prefixing with an underscore for things which should be private, which this probably also should be.